2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1037,6 +1037,58 @@ Questa attività consiste nei seguenti compiti:
 
 = Processi tecnici
 
+== Processo di analisi della missione
+
+_Tailored conformance on ISO/IEC/IEEE 12207:2017 clause 6.4.1 per ISO/IEC/IEEE 12207:2017 Annex A_
+
+=== Scopo
+
+Il processo di analisi della missione definisce i problemi e le opportunità dai quali scaturisce il progetto, caratterizza lo spazio delle soluzioni e determina una classe di soluzione preferita.
+
+=== Strategia di identificazione e analisi della missione
+
+==== Sistemi e servizi abilitanti
+Gli strumenti di comunicazione adottati dal gruppo sono descritti in TODO.
+
+==== Opportunità
+Si analizzano i problemi e le opportunità così da ottenere una visione d'insieme dello status quo. L'obiettivo dell'analisi è la comprensione del contesto e delle cause di problemi e opportunità, e quindi del Capitolato.
+
+Si identifica l'ambito del Capitolato attraverso la definizione di:
+- difetti e _pain point_ delle soluzioni pre-esistenti, se disponibili;
+- soluzioni alternative, con vantaggi e svantaggi rispetto alle soluzioni pre-esistenti, se disponibili;
+- contesto tecnologico di applicazione;
+- tipologia di utenza attesa;
+- destinazione d'uso del prodotto finale.
+
+La sintesi di bisogni e requisiti avviene nel contesto del Processo di definizione di bisogni e requisiti degli stakeholder (TODO).
+
+==== Classi di soluzione
+Si identificano classi di soluzione che possano sfruttare le opportunità e risolvere i problemi individuati.
```

Comment:
per andare a capo e creare nuovi paragrafi una \ a fine paragrafo altrimenti l'indentazione impazzisce

Reply:
si può andare a capo anche con 2 "invio"... non credo cambi molto, ma potenzialmente in questo modo si evita di aggiungere caratteri "\"... in caso manda screen di cosa succede all'indentazione

Reply:
**Senza `\`**:
![Screenshot_20240103_111152](https://github.com/Error-418-SWE/Documenti/assets/2914344/1b652c37-3c36-4bd2-8be4-fcdfc708618d)

**Con `\`**:
![Screenshot_20240103_111247](https://github.com/Error-418-SWE/Documenti/assets/2914344/f34af2fa-7665-4057-bb9a-662a6f661209)

Personalmente non mi piace nessuna delle due soluzioni ahahah
La prima, come fa notare Antonio, sminchia l'indentazione (avevo già controllato e secondo Typst "it's not a bug, it's a feature").
La seconda introduce caratteri che rendono meno leggibile il sorgente e riduce l'interlinea perché tecnicamente compatta le linee ad un singolo paragrafo.

Se mettessi mano al template per eliminare lo spazio a inizio riga secondo voi andrebbe bene lo stesso?

Reply:
Potrebbe essere una valida opzione… prima di apportare modifiche al template peró ti chiederei di mostrare un eventuale screen di come cambierebbe se possibile🙏

Reply:
**Before**:
![wi](https://github.com/Error-418-SWE/Documenti/assets/2914344/c058896f-2d5b-41dd-8508-f4d325b888a9)

**After**:
![wo](https://github.com/Error-418-SWE/Documenti/assets/2914344/4eaa47d2-f5c2-4950-bc73-e15cdfb2c346)

Ho rimosso l'identazione dei paragrafi *consecutuivi*. Altri elementi non sono affected

Reply:
Se accettata, vedi [DOC-301] (PR #140)

Reply:
io ragazzi questa cosa dell'andare a capo senza \ l'ho sempre segnata come errore in pr perché vedere un paragrafo indentato diversamente da un altro senza motivo mi urtava, modificando il template cosa abbiamo cambiato? che adesso si può andare a capo senza \?

giovanni non prendere questo commento con tono arrogante, solo mi domando, ha senso tutto questo lavoro per non mettere un \?